week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -13,9 +13,9 @@ function mood() {
 
 function greaterThan10() {
   let num = 10;
-  let isBigEnough;
+  let isBigEnough = 12;
 
-  if (isBigEnough) {
+  if (isBigEnough >= num) {
```

Comment:
Given its name, the variable `isBigEnough` implies it represents a boolean (the response to "is it big enough?").

Without changing the `if` condition, how would change that `let isBigEnough` to contain the right value based on `num`?